packages/sdk/src/zora/index.ts

```diff
@@ -1,3 +1,7 @@
 import * as Addresses from "./addresses";
+import { Exchange } from "./exchange";
+import { ModuleManager } from "./moduleManager";
```

Comment:
Let's rename this to `module-manager` to be consistent with the file naming across the repo.